### PR TITLE
EISN Low Level Sectorisation

### PR DESCRIPTION
**Pull request by Fechulo**
## Description of changes
Added Shannon Low Level Sector now possible thanks to 1.4.0

## Reason and motivation
More accurately represents low level sector (which does not cover EIDW top-down)

## Approved contributor
- [x] I am on the approved contributers list
- [ ] I have sent an request by email to get approved
- [ ] Someone on the approved contributer list will review this request

### CODE REVIEW

File: VATSpy.dat [hunk 1]
```diff
@@ -17965,8 +17965,9 @@ EGYP|Mount Pleasant (Monte Agradable)|ISLAND|EGYP
 EHAA|Amsterdam||EHAA
 EHMC|Dutch Mil (Up to FL245, including N and E)||EHMC
 EHML|Bandbox CRC (Military)||EHAA
-EIDW|Dublin ACC (Up to FL245) - Shannon||EIDW
-EISN|Shannon||EISN
+EIDW|Dublin ACC (Up to FL245) - Shannon|EIDW|EIDW
+EISN|Shannon ACC - Shannon|EISN|EISN
```
Comment (Adrian2k) [inline VATSpy.dat:17969]: Not nessecerily to add alias on these two since they match the FIR.

Comment (Fechulo) [reply VATSpy.dat]: Fixed

**Review by Adrian2k — CHANGES_REQUESTED** [review]
image

It seems to be causing issues for me when trying to open it..

Also by the looks of the coordinates - it does not seem to be alligned with excisting borders

**Comment by Fechulo** [comment]
@Adrian2k Apologies, should all be fixed now. As for the coordinates, exporting it to geojson added extra decimal places but for all intents and purposes the points are the same.

**Review by NelisV — CHANGES_REQUESTED** [review]
This still needs some work before we can approve, borders are not aligned in various places:
image
image

**Review by Adrian2k — APPROVED** [review]
Preconditional approval - Awaiting @NelisV review

**Review by NelisV — APPROVED** [review]

**Pull request merged**